.github/workflows/test.yml

```diff
@@ -49,23 +49,55 @@ jobs:
           - toxenv: py39-dj40
             python-version: 3.9
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
 
       - name: Set up Python
-        uses: actions/setup-python@v1
+        uses: actions/setup-python@v2
```

Comment:
Should you pull in the latest here? Which is 2.3.1 https://github.com/actions/setup-python/releases

Reply:
Same as a above, the [usage](https://github.com/actions/setup-python#usage) does not suggest using more than `actions/setup-python@v2`.

Reply:
Ah yea looking at the raw log I see it checks out SHA f38219332975fe8f9c04cca981d674bf22aea1d3 (https://github.com/actions/setup-python/commit/f38219332975fe8f9c04cca981d674bf22aea1d3), which is the same as the latest release https://github.com/actions/setup-python/releases/tag/v2.3.1

Hmm I guess we should strip off the minor and patch versions I added to these elsewhere some time 😬

Reply:
Can confirm that `@v#` means "anything in the major version # series":

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsuses